packages/wdio-repl/package.json

```diff
@@ -22,6 +22,8 @@
   "bugs": {
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
+  "types": "./build/index.d.ts",
```

Comment:
Is this build dir also what is available when the package is installed? Because if not, then that part has to be removed from the path

Reply:
Yes, the NPM package will only contain a `build` dir.